common-requests/use.md

```diff
@@ -1,8 +1,157 @@
+* `0.agreement_execution_start_time` - When the edge node began downloading the container image(s)
+
+Once you see a value for `agreement_execution_start_time`, you should expect to be able to run `docker ps` to see the workloads starting.
+
+## Other things to try
+
+To see the logs of the sample "helloworld" service installed by default, try `hzn service log -f ibm.helloworld`.
+
+To view all of the steps logged in the process of negotiating an agreement, type `hzn eventlog list`.
+
+And to view the node's policy that triggered deployment of the "helloworld" service, try `hzn policy list`.
+
+## One last thing ...
+
+We've been using the CLI to communicate with the Anax agent.  There's a whole other set of commands to run that have you communicating with the Exchange.  To get those working, you'll need to set two environment variables.  To do so, you'll need one important bit of information from the all-in-one installation summary message.  Hopefully you saved that information.  It should look something like this:
+
+``` text
+----------- Summary of what was done:
+  1. Started Horizon management hub services: agbot, exchange, postgres DB, CSS, mongo DB
+  2. Created exchange resources: system org (IBM) admin user, user org (myorg) and admin user, and agbot
+     - Exchange root user generated password: 1234567890
+     - System org admin user generated password: AbcDEfg
+     - Agbot generated token: Abc123YZ
+     - User org admin user generated password: XLmdsg236
+     - Node generated token: uingtw398J
+     Important: save these generated passwords/tokens in a safe place. You will not be able to query them from Horizon.
+  3. Installed the Horizon agent and CLI (hzn)
+  4. Created a Horizon developer key pair
+  5. Installed the Horizon examples
+  6. Created and registered an edge node to run the helloworld example edge service
+For what to do next, see: https://github.com/open-horizon/devops/blob/master/mgmt-hub/README.md#all-in-1-what-next
+```
+
+Using the "User org admin user generated password" from the message above, export the following two environment variables with the substitution shown below:
+
+``` shell
+export HZN_ORG_ID=myorg
+export HZN_EXCHANGE_USER_AUTH=admin:[User org admin user generated password]
+```
+
+Once you've done that, you should be able to connect to the Exchange with the CLI.  Try `hzn exchange status` or `hzn exchange node list` to confirrm that it is working with no errors.  Also, `hzn exchange user list` will show you what user account you are currently using, and if it is successfully authenticated.
+
+Here are three commands you can now run:
+
+* View the example edge services: `hzn exchange service list IBM/`
+* View the example patterns: `hzn exchange pattern list IBM/`
+* View the example deployment policies: `hzn exchange deployment listpolicy`
+
+## Want some more?
+
+To view other commands to run, try exploring by using the following pattern "hzn [command] (sub-command) -h".
```

Comment:
Use the hzn [command] (sub-command) -h pattern to explore other commands.

(Should we show any examples?)